### ISSUE TITLE
Proposal: Disable benchmark runs on Travis

### ISSUE THREAD

**Issue by djspiewak** [functional-streams-for-scala/fs2#779]
The Travis public servers are quite unreliable in terms of performance.  They don't guarantee availability of resources, and my understand from their architecture is they *can't* guarantee resources on the public cluster.  We (Verizon) ran into this as a problem when doing our latest batch of open sourcing, since moving projects from our private Travis to the public one resulted in a >10x increase in build times.  (Travis itself is wicked fast btw, it's just that their public infrastructure is very throttled and under very heavy load)

To the broader point…  Running benchmarks on Travis is basically pointless, since the results are not going to be sanely comparable from run-to-run.  If we really want to do benchmarking on every build (which would be fantastic!), we need to spin off a separate instance somewhere which gives us predictable performance.  Obviously, that takes some money/sponsorship to achieve, so I'm going to leave that discussion aside for the moment.

tl;dr: Calling `jmh:run` in Travis builds is just burning CPU and slowing down releases for almost no useful data.

**Comment by mpilquist** [comment]
👍 Completely agree

**Comment by pchiusano** [comment]
👍
